apps/server/src/modules/files/job/sync-filerecords.console.ts

```diff
@@ -0,0 +1,62 @@
+import { EntityManager, ObjectId } from '@mikro-orm/mongodb';
+import { File, FileRecord, FileRecordParentType } from '@shared/domain';
+import { FileFilerecord } from '@shared/domain/entity/file_filerecord.entity';
+import { TaskRepo } from '@shared/repo';
+import { Logger } from '@src/core/logger';
+import { Command, Console } from 'nestjs-console';
+
+@Console({ command: 'sync-filerecords' })
+export class SyncFilerecordsConsole {
+	constructor(private taskRepo: TaskRepo, private em: EntityManager, private logger: Logger) {}
+
+	@Command({ command: 'tasks [batchSize]' })
+	async syncFilerecordsForTasks(batchSize = 50) {
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
interface Options {
	batchSize: number = 50;
}
async syncFilerecordsForTasks(option: Option) {

const tasksToSync: TaskToSync[] = await this.taskRepo.getTasksToSync(option.batchSize)

Should make it type save